sw/device/rom_exts/manifest.h

```diff
@@ -0,0 +1,220 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_ROM_EXTS_MANIFEST_H_
+#define OPENTITAN_SW_DEVICE_ROM_EXTS_MANIFEST_H_
+
+// This file is the "ROM_EXT manifest header template".
+// It has been generated by running the command:
+// $ util/rom-ext-manifest-generator.py.
+
+// Header Extern Guard (so header can be used from C and C++)
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * ROM_EXT manifest field manifest_identifier offset from the base.
+ */
+#define ROM_EXT_MANIFEST_IDENTIFIER_OFFSET 0u
+
+/**
+ * ROM_EXT manifest field manifest_identifier size in bytes and words.
+ */
+#define ROM_EXT_MANIFEST_IDENTIFIER_SIZE_BYTES 4u
+#define ROM_EXT_MANIFEST_IDENTIFIER_SIZE_WORDS 1u
+
+/**
+ * ROM_EXT manifest field image_signature offset from the base.
+ */
+#define ROM_EXT_IMAGE_SIGNATURE_OFFSET 8u
+
+/**
+ * ROM_EXT manifest field image_signature size in bytes and words.
+ */
+#define ROM_EXT_IMAGE_SIGNATURE_SIZE_BYTES 384u
+#define ROM_EXT_IMAGE_SIGNATURE_SIZE_WORDS 96u
```

Comment:
Do we want these to be dec or hex?

Reply:
Not sure to be honest, I kind of felt that size is nice to have in dec and offsets in hex, but really I don't have a strong preference. Happy to change it either way.

Reply:
These are generated, so it doesn't really matter, and we've used decimal everywhere else (reggen, topgen).

Reply:
Keeping as ints.